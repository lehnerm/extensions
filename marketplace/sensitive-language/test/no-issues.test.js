'use strict';

const React = require('react');
const renderer = require('react-test-renderer');

function mockComponent (componentClassName) {
  return ((props) => {
    const className = props.className ? `${componentClassName} ${props.className}` : componentClassName;

    return (<div className={className} {...props}>{props.children}</div>)
  });
}

jest.mock('@contentful/forma-36-react-components', () => ({
  Icon: mockComponent('icon'),
  Typography: mockComponent('typography'),
  Subheading: mockComponent('subheading'),
  Paragraph: mockComponent('paragraph')
}));

const {NoIssues} = require('../src/no-issues.js');

test('NoIssues: renders correctly', function () {
  const tree = renderer
  .create(<NoIssues/>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
