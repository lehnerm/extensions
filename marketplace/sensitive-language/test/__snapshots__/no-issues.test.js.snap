// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoIssues: renders correctly 1`] = `
<div
  className="typography"
>
  <div
    className="subheading"
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    <div
      className="icon"
      color="positive"
      extraClassNames="f36-margin-right--xs"
      icon="CheckCircle"
    />
    No issues found
  </div>
  <div
    className="paragraph"
    style={
      Object {
        "marginBottom": 0,
      }
    }
  >
    All checked files seem to be free on insenstive language. Good job!
  </div>
</div>
`;
